definitions/private-law/json/CaseField/RespondentSolicitor/CaseDocuments/CaseField.json

```diff
@@ -94,9 +94,55 @@
     "LiveFrom": "12/06/2023",
     "CaseTypeID": "PRLAPPS",
     "ID": "respondentEc8",
-    "Label": "Respondent C8 documents",
+    "Label": "Respondent C8 document",
     "FieldType": "ResponseDocuments",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+
+  {
+    "LiveFrom": "20/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "respondentAc8Documents",
+    "Label": "Respondent C8 document",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "ResponseDocuments",
+    "SecurityClassification": "Public"
+  },
```

Comment:
Once we have this new field, we need to get rid of old fields I think. confirming with BA.